upload-api/tables/rate-limit.js

```diff
@@ -0,0 +1,75 @@
+import {
+  DeleteItemCommand,
+  DynamoDBClient, PutItemCommand, QueryCommand,
+} from '@aws-sdk/client-dynamodb'
+import { marshall, unmarshall } from '@aws-sdk/util-dynamodb'
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
we're currently using `nanoid` in this project for id generation https://github.com/ai/nanoid

Reply:
oh awesome, will update!